### DISCUSSION TITLE
Importing ProPresenter Libraries

### DISCUSSION THREAD

**Discussion by jwamoo** [ChurchApps/ChurchAppsSupport#309] [FreeShow] [answered]
I love your app and what you are doing with it.
It looks good and though I am yet to use it in church, I believe it would be a fantastic presentation app.
I have a challenge though, I have been trying with no success to import the songs library of Propreseter. 
I realized that FreeShow is looking for files with extensions .pro6 and .pro7 but my song files have just .pro.
I don't know if I am the one not doing something right and need help with it or if there is a bug that needs fixing.
Could you please help?
Thank you.

**Comment by vassbo** [comment, marked as answer]
Hi, thank you so much. I would like to help.

I think it's just .pro because it's an older version, with another format "inside" the file.
If you could send over a couple of the .pro files on mail: dev@freeshow.app, that would be nice. So I can add support for those files too. 😊

**Reply by jwamoo** [reply]
Thanks for that prompt response. 
I have sent you a mail with some of the .pro file attachments from my Gmail account.
Please let me know if this helps and or if you need anything else.
Blessings!
